qwp_integration_test.go

```
@@ -0,0 +1,703 @@
+	u, _ := url.Parse("http://" + qwpTestAddr)
+	u.Path = "/exec"
+	params := url.Values{}
+	params.Add("query", "DROP TABLE IF EXISTS '"+tableName+"';")
+	u.RawQuery = params.Encode()
+
+	resp, err := qwpTestHTTPClient.Get(u.String())
+	if err != nil {
+		t.Logf("warning: could not drop table %q: %v", tableName, err)
+		return
+	}
+	resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)